html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="Email">Email</label>
+    <input type="text" id="Email">
+
+    <div>
+      <label for="Why">Why is coding cool?</label>
+      <div>
+        <select name="reason" id="Why">
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+    </div>
+
+    <div>Favorite Language</div>
+    <div>
+      <input type="radio" name="favorite" id="html" value="html">
+      <label for="html">HTML</label>
+      <input type="radio" name="favorite" id="css" value="css">
+      <label for="css">CSS</label>
+      <input type="radio" name="favorite" id="javascript" value="Javascript">
+      <label for="javascript">Javascript</label>
+    </div>
+
+    <div>
+      <label for="signup">Would you like to signup for more information?</label>
+      <div>
+        <input type="checkbox" id="signup" name="signup" value="signup">
```

Comment:
I think we are missing `Sign me up!` label for the checkbox :)
Then the `Would you like to signup...` can become a `<p>`